novawallet/Common/Services/ChainRegistry/ConnectionPool/ConnectionFactory.swift

```diff
@@ -123,20 +119,18 @@ extension ConnectionFactory: ConnectionFactoryProtocol {
     }
 
     private func createConnection(
-        urlModels: [ConnectionTLSSupport],
+        urlModels _: [ConnectionCreationParams],
```

Comment:
Seems like we can remove it since no use. Or it's better to remove `urls` argument so we can use url from provided models.